christfellowshipchurch/apollos-api
# Enable ActionListFeature and Rearrange Sisterhood Features

### About
This PR enables the `ActionListFeature` for the So Good Sisterhood page and rearranges the features to the order that was requested.

### _🛑  DO NOT MERGE UNTIL THE MORNING OF NOV 1ST 🛑_ 

### Test Instructions
* Run API locally with `CONTENT=stage` (to see the new podcast).
* Run web-app-v2 locally with API
* go to `/so-good-sisterhood` and see the `ActionListFeature` in the expected place.

### Closes Tickets
[CFDP-1775]


[CFDP-1775]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1775?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
